examples/ads.amp.html

```diff
@@ -401,6 +402,13 @@ <h2 id="atomx">Atomx</h2>
       data-id="1234">
   </amp-ad>
 
+  <h2 id="bidtellect">Bidtellect</h2>
```

Comment:
I can't see a rendered ad example. I get 

> Bidtellect: The placement 815676411 is missing some params.

Please check. Also I thought `data-t` and `data-pid` are required, all other params are optional, is it not the case?